aws-ts-ecr-cache/index.ts

```diff
@@ -0,0 +1,96 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+
+const pulumiConfig = new pulumi.Config();
+const enableDocker = pulumiConfig.getBoolean("enableDocker") ?? true;
```

Comment:
I think just to reduce inputs, I would get rid of the "enable" boolean configs, and make the conditional resources conditional on whether there is both a username and password for that service in the config - and also, fail the update if there are no username/password combinations

Reply:
or at least, I would make the "enable" booleans default to false